cf/mixin/propertiesdatabounds.py

```diff
@@ -3758,6 +3758,71 @@ def persist(self, bounds=True, inplace=False):
             inplace=inplace,
         )
 
+    @_inplace_enabled(default=False)
+    def rechunk(
+        self,
+        chunks=_DEFAULT_CHUNKS,
+        threshold=None,
+        block_size_limit=None,
+        balance=False,
+        bounds=True,
+        interior_ring=True,
+        inplace=False,
+    ):
+        """Change the chunk structure of the data.
+
+        .. versionadded:: TODODASKVER
+
+        .. seealso:: `cf.Data.rechunk`
+
+        :Parameters:
+
+            {{chunks: `int`, `tuple`, `dict` or `str`, optional}}
+
+            {{threshold: `int`, optional}}
+
+            {{block_size_limit: `int`, optional}}
+
+            {{balance: `bool`, optional}}
+
+            bounds: `bool`, optional
+                If False then do not rechunk any bounds. By default
```

Comment:
As with a previous comment:

```suggestion
                If True (the default) rechunk the bounds.
```

Reply:
Fixed with https://github.com/NCAS-CMS/cf-python/pull/543/commits/cfa222288bf89843df4fe5bbc39d251edbf587d0